lib/inventory_refresh/graph.rb

```diff
@@ -37,7 +37,7 @@ def to_graphviz(layers: nil)
         s << "  #{node_names[from]} -> #{node_names[to]};"
       end
       s << "}"
-      s.join("\n") + "\n"
+      "#{s.join("\n")}\n"
```

Comment:
I totally saw this going with using "<<" instead of string interpolation

Reply:
Same

Reply:
I bet if `<<` were used, then it wouldn't have tried to autoconvert, much line the previous line.  @agrare what do you like better?